### PR TITLE
Select, Save and Restore Gateway Assignment Settings

### PR DESCRIPTION
**Pull request by npsp-reedestockton**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by npsp-reedestockton** [comment]
@lparrott @voduyemi I've tagged you for review. Would greatly appreciate your comments. Jest tests will be addressed in a separate work item.

**Comment by npsp-reedestockton** [comment]
Tracking [W-11564984](https://gus.lightning.force.com/lightning/r/a07EE000013ZMecYAG/view), [W-11565009](https://gus.lightning.force.com/lightning/r/a07EE000013ZMYYYA4/view), [W-11632653](https://gus.lightning.force.com/lightning/r/a07EE000014h7CvYAI/view)

**Review by lparrott — APPROVED** [review]
Nice work! I'm hoping to take a more thorough look, but wanted to provide initial thoughts.

File: force-app/main/default/classes/GE_GiftEntryController.cls [hunk 1]
```diff
@@ -1257,6 +1268,51 @@ public with sharing class GE_GiftEntryController {
         return (String)batches[0].get(UTIL_Currency.CURRENCY_ISO_CODE_FIELD);
     }
 
+    @AuraEnabled
+    public static String getGatewaysFromElevate() {
```
[the remark this answers is not in the document]
Comment (npsp-reedestockton) [reply force-app/main/default/classes/GE_GiftEntryController.cls:1272]: Good callout. Three of these can be combined, but encrypt, decrypt and get gateways need to remain separate calls. I'll combine the three that can be combined.

File: force-app/main/default/lwc/geTemplateBuilderWidget/geTemplateBuilderWidget.html [hunk 2]
```diff
@@ -16,12 +16,17 @@
...
                     </lightning-layout-item>
-                </template>                
+                </template>
                 <template if:true={isTokenizeCard}>
                     <lightning-layout-item size="12">
-                        <p class="slds-text-body_regular slds-text-title_bold widget-text">
-                            {CUSTOM_LABELS.geTextFieldBundlePaymentMethod }
-                        </p>
+                        <label class="slds-text-body_regular widget-label slds-p-bottom_xx-small">{uiTitle}</label>
+                        <p class="slds-text-body_regular widget-text">{CUSTOM_LABELS.commonCreditNumber}</p>
```
[the remark this answers is not in the document]
Comment (npsp-reedestockton) [reply force-app/main/default/lwc/geTemplateBuilderWidget/geTemplateBuilderWidget.html:23]: The UX calls for listing the fields included in the bundle, but they aren't ours. I don't see a better alternative, but I'm open to suggestions.

[the remark this answers is not in the document]
Comment (ErinWiedemer) [reply force-app/main/default/lwc/geTemplateBuilderWidget/geTemplateBuilderWidget.html:23]: Would it help to add the term "Elevate" to the intro sentence? Like: "This Elevate field bundle can include the following fields:"?

[the remark this answers is not in the document]
Comment (npsp-reedestockton) [reply force-app/main/default/lwc/geTemplateBuilderWidget/geTemplateBuilderWidget.html:23]: @ErinWiedemer I'm not sure that helps. The fields are still listed there without any qualification as to whether or not users should be able to find them in their salesforce org. If we really think this is important to address, I think the solution is either docs or add verbiage to the widget explaining that these fields aren't ours.

File: force-app/main/default/classes/GE_GiftEntryController.cls (hunk 1 above)
[the remark this answers is not in the document]
Comment (npsp-reedestockton) [reply force-app/main/default/classes/GE_GiftEntryController.cls:1272]: @lparrott PR feedback implemented to combine three of these calls.

**Review by voduyemi — COMMENTED** [review]
Looks good so far, just had a few comments!

File: force-app/main/default/lwc/geTemplateBuilderWidget/geTemplateBuilderWidget.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (ErinWiedemer) [reply force-app/main/default/lwc/geTemplateBuilderWidget/geTemplateBuilderWidget.html:23]: @npsp-reedestockton I can definitely mention it in docs. But I'd be happy to go with what Grace thinks is best. Would an "i" bubble with explanatory text work?

**Comment by npsp-reedestockton** [comment]
@voduyemi Implemented your PR feedback. All excellent suggestions. Thanks!

File: force-app/main/default/lwc/geTemplateBuilderWidget/geTemplateBuilderWidget.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (lparrott) [reply force-app/main/default/lwc/geTemplateBuilderWidget/geTemplateBuilderWidget.html:23]: Leaving it alone is also an option, we can make changes if we receive feedback that it's confusing.

[the remark this answers is not in the document]
Comment (ErinWiedemer) [reply force-app/main/default/lwc/geTemplateBuilderWidget/geTemplateBuilderWidget.html:23]: Let's leave it, and I'll add info to the docs article. If we get word that this is confusing people, we can revisit UI work and tweak the docs.

**Review by voduyemi — APPROVED** [review]
Thanks! @npsp-reedestockton. Looks good to me.

**Pull request merged**